docs/docker.md

```diff
@@ -8,13 +8,141 @@ Start by downloading and installing Docker CE for your platform:
 * [Windows](https://docs.docker.com/docker-for-windows/install/)
 * [Linux](https://docs.docker.com/install/)
 
+Optionally, [docker-compose](https://docs.docker.com/compose/install/) should be installed and available to follow the docker quick start guide.
```

Comment:
```suggestion
Optionally, [docker-compose](https://docs.docker.com/compose/install/) should be installed and available to follow the docker quick start guide below.
```

-- or a link to `### Docker quick start`

Reply:
yes - link is better than "below"... ;)